Aptoide/aptoide-client-v8
# [ASV-952] Fixed wrong style buttons

…tep_two.

**What does this PR do?**

   fixed wrong style buttons on profile step one and step two.

**Database changed?**

   No

**Where should the reviewer start?**

- [x] fragment_profile_step_one.xml
- [x] fragment_profile_step_two.xml

**How should this be manually tested?**

  Run the app and create a user. Check the buttons after inserting name and date of birth. Click on more info and check the buttons.

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-950


**Code Review Checklist**
